cmd/helm/init.go

```diff
@@ -42,10 +47,21 @@ const (
 type initOptions struct {
 	skipRefresh         bool   // --skip-refresh
 	stableRepositoryURL string // --stable-repo-url
+	pluginsFilename     string // --plugins
 
 	home helmpath.Home
 }
 
+type pluginsFileEntry struct {
+	Name    string `json:"name,omitempty"`
+	Url     string `json:"url"`
```

Comment:
`Url` should be `URL`

Reply:
Whoops, must've missed this in the linter. Fixed.